libfwupdplugin/fu-bios-settings.c

```diff
@@ -555,6 +555,33 @@ fu_bios_settings_get_all(FuBiosSettings *self)
 	return g_ptr_array_ref(self->attrs);
 }
 
+/**
+ * fu_bios_settings_get_pending_values:
+ * @self: a #FuBiosSettings
+ *
+ * Gets all the attributes in the object.
+ *
+ * Returns: (transfer container) (element-type FwupdBiosSetting): attributes
+ *
+ * Since: 1.8.4
```

Comment:
1.9.4, no?

Reply:
Thank you for reviewing.

It should be 1.9.4. I'll correct this.